template/.babelrc

```diff
@@ -1,7 +1,8 @@
 {
   "presets": [
   	"react",
-  	['es2015', {'modules': false}]
+  	['es2015', {'modules': false}],
+    "minify"
```

Comment:
The indentation looks funky here for these lines.

Reply:
- [ ] Fix indentation